### PR TITLE
3.16.0

### PR DESCRIPTION
**Pull request by franmomu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ a release.
 ---
 
 ## [Unreleased]
+
+## [3.16.0]
 ### Deprecated
 - Support for `doctrine/annotations` has been deprecated and will be removed in 4.0.
```
Comment (phansys) [inline CHANGELOG.md:24]:
```suggestion
- Support for defining mapping information from annotations has been deprecated and will be removed in 4.0, use PHP attributes mapping instead.
```

**Review by phansys — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (phansys) [reply CHANGELOG.md:24]: BTW, I think we should also update this statement:
https://github.com/doctrine-extensions/DoctrineExtensions/blob/ca713c5ddd26e0f7eb94b95fa1e863efcc1bbd7d/README.md?plain=1#L64

Comment (franmomu) [reply CHANGELOG.md:24]: sure, let me change it before releasing, anything else you think we should take a look at?

Comment (phansys) [reply CHANGELOG.md:24]: No @franmomu, that's all.

Thank you so much!

**Pull request merged**